srbhr/Resume-Matcher
# Added cohere to requirements.txt to fix import error

## Pull Request Title
Added cohere to requirements.txt to fix import error

## Related Issue
Fixes #257

## Description
This pull request adds `cohere` to the `requirements.txt` file to resolve the import error that occurs when the module is not found.

## Type



## Proposed Changes


- Added `cohere` to `requirements.txt`

## Screenshots / Code Snippets (if applicable)

![image](https://github.com/srbhr/Resume-Matcher/assets/84693614/94bfb70e-974d-4f2f-8b03-feaabf0d53e9)

## How to Test


1. Clone the repository
2. Install the dependencies using `pip install -r requirements.txt`
3. Verify that the `cohere` module can be imported without errors

## Checklist



## Additional Information

N/A
